internal/feed/newznab.go

```diff
@@ -75,8 +75,16 @@ func (j *NewznabJob) process(ctx context.Context) error {
 	}
 
 	releases := make([]*domain.Release, 0)
-
+	now := time.Now()
 	for _, item := range items {
+		if j.Feed.MaxAge > 0 {
+			if item.PubDate.After(time.Date(1970, time.April, 1, 0, 0, 0, 0, time.UTC)) {
+				if !isNewerThanMaxAge(j.Feed.MaxAge, item.PubDate.Time, now) {
+					return nil
```

Comment:
Shouldn't this just be `continue` rather than return?

Reply:
most certainly